.github/ISSUE_TEMPLATE/test_case.md

```
@@ -0,0 +1,57 @@
+
+**Environment:**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Additional context**
+Add any other context about the test here.  
+ 
+ User story links E.g.: "User story #100 "
+
+**Test Execution**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)